modules/sdk/src/main/scala/org/tessellation/sdk/config/types.scala

```diff
@@ -97,7 +97,9 @@ object types {
 
   case class TrustStorageConfig(
     ordinalTrustUpdateInterval: NonNegLong,
-    ordinalTrustUpdateDelay: NonNegLong
+    ordinalTrustUpdateDelay: NonNegLong,
+    seedlistInputBias: Double,
+    seedlistOutputBias: Double
```

Comment:
Is the domain of these two variables the same as `Double`?

Reply:
Per @ryle-goehausen , these values are to stay `Double`s.

Reply:
@ryle-goehausen can you explain why? we use refined types in the whole codebase

Reply:
We can potentially add a new refined type here, but these are part of the model configuration -- which so far has no hyperparameters exposed to the user. This should in fact be part of a model hyperparameter configuration (see other parameters in the model.) Since we're fixing all the hyperparameters at compile time, it's assume they are validated here by construction. The actual 'validator' for this is human oversight, since it's not possible to ever input a value here. And a valid value doesn't really have the same refinement as prior (since negative values would be meaningless to add here.) If we added supported for model configuration at runtime (i.e. dynamic configs,) then yeah, we can create a refined type for this then if necessary.

I'd just leave it as is, considering all the other parameters we're using in model are double typed.